### DISCUSSION TITLE
There is no place to manage/replace/upgrade vehicles based on routes view

### DISCUSSION THREAD

**Discussion by Miszkaaaa** [voxeltycoon/issues#776] [UI&UX] [answered]
**Problem:**
To be able to manage/see list of assigned vehicles to a route you need to:
1. Check the route which is interesting
2. Open a station
3. Try to manage vehicles from that place

![image](https://user-images.githubusercontent.com/10045272/118519960-e18c0180-b739-11eb-918b-7fb39e9007a2.png)

**Solution:**

Add a tab for managing routes

![image](https://user-images.githubusercontent.com/10045272/118522875-d9819100-b73c-11eb-822a-232c5458a4f6.png)

Additionally:
1. There could be a switch to put all vehicles to a garage from a route.
2. There could be an upgrade button/template to replace/upgrade multiple vehicles/trains

**Comment by victorleblais** [comment, marked as answer]
A proper route manager is planned, however, you can filter the Fleet Manager window to only show one (or more) routes by pressing this button:
![image](https://user-images.githubusercontent.com/4365075/118854235-e1743900-b8d4-11eb-86ff-b739eb754750.png)

**Reply by Miszkaaaa** [reply]
Thanks for the tip!
